pages/mesosphere/dcos/services/hive-metastore/1.0.0-3.0.0/supported-databases/index.md

```diff
@@ -0,0 +1,105 @@
+---
+layout: layout.pug
+navigationTitle: Supported Databases
+excerpt: Supported Databases
+title: Getting Started
+menuWeight: 25
+model: /mesosphere/dcos/services/hive-metastore/data.yml
+render: mustache
+---
+
+## Supported Databases
+
+The DC/OS {{ model.techName }} supports the following relational databases (RDBMS):
+ - Derby
+ - MySQL
+ - Percona-MySQL
+
+### Install the {{ model.techShortName }} with Derby
+By default, {{ model.techName }} configuration is set to the default embedded Derby database for development usage. However, it is not intended for use beyond simple testing.
+
+```
+dcos package install {{ model.packageName }}
+```
+
+
+### Install the {{ model.techShortName }}  with MySQL
+The {{ model.techShortName }} can also be backed by a MySQL database. This is recommended for production use.
+
+- Users must deploy MySQL:
+
+   ```
+   dcos package install mysql
```

Comment:
Should we include/mention the MySQL version that we're currently using for our deployments? As we know there are other versions of MySQL in catalog and we should recommend which version is compatible with our HiveMS installation.

Reply:
We could but the latest version in catalogue is already 1.5-2 years old and is the version we test with in CI/soak. Later on we might have to test explicit mysql versions (or even update the package ourselves) but for now, this works.